src/libs/actions/Policy/Tag.ts

```diff
@@ -618,6 +618,7 @@ function setPolicyTagsRequired(policyID: string, requiresTag: boolean, tagListIn
                 value: {
                     [policyTag.name]: {
                         required: policyTag.required,
+                        errors: ErrorUtils.getMicroSecondOnyxError('workspace.tags.genericFailureMessage'),
```

Comment:
As in the above screenshot we have another (already existing bug) in the styling of the second error message. Can you fix that here if it's not much work? (otherwise we can do it in a follow up)

Reply:
Sorry for the delay, I thought I replied to this already 😅. The onyx keys are case sensitive it's `policyTags_`

```js
Onyx.merge('policyTags_CE194D2E3EC60BA3', {"Country": {errorFields: {required: {0: "error here"}}}})
```